olm/eclipse-che-preview-openshift/deploy/olm-catalog/eclipse-che-preview-openshift/7.10.0/eclipse-che-preview-openshift.crd.yaml.diff

```diff
@@ -0,0 +1,16 @@
+--- /home/tolusha/gocode/src/github.com/eclipse/che-operator/olm/eclipse-che-preview-openshift/deploy/olm-catalog/eclipse-che-preview-openshift/7.9.0/eclipse-che-preview-openshift.crd.yaml	2020-02-26 09:42:38.512787498 +0200
++++ /home/tolusha/gocode/src/github.com/eclipse/che-operator/olm/eclipse-che-preview-openshift/deploy/olm-catalog/eclipse-che-preview-openshift/7.10.0/eclipse-che-preview-openshift.crd.yaml	2020-03-15 08:48:33.615432343 +0200
+@@ -293,6 +293,13 @@
+                     config map from other CR fields, then the value defined in the
+                     `customCheProperties` will be used instead.
+                   type: object
++                serverTrustStoreConfigMapName:
```

Comment:
Are we sure we really need that in the OpenShift case ? Normally, in OpenShift + TLS + self-signed certs, the self-signed cert is already extracted automatically, and added to the che-server. How does this addition relates to the already-existing self-signed cert management ? Is it compatible with it ?

I'm OK that it's not really related to this PR specifically, but more to a more general question.